frontend/src/sections/WhatIsDjango/WhatIsDjango.tsx

```diff
@@ -43,7 +46,7 @@ const WhatIsDjango = () => {
               contribute to the growth of the Django ecosystem in India.
               </p>
               <div className='z-20 pl-8'>
-                <Button>Learn More</Button>
+                <Button onClick={handleClick}>Learn More</Button>
```

Comment:
Can't we use next/link ?

Reply:
Of course, We can put the button within anchor tag and put the href to it.

Reply:
This is better, allows more room for future functionality for this section & button.